ocaml/xapi/dune

```diff
@@ -146,6 +146,7 @@
     sexplib0
     sexpr
     sha
+    ssl
```

Comment:
We usualy avoid linking `ssl` into XAPI and use `stunnel` in client mode instead.
OpenSSL is a lot of C code, and I don't particularly trust our modified version of it, it could lead to crashes/memory corruption. At least with `stunnel` it is isolated, if we bring it into XAPI, it could corrupt XAPI's memory.

Reply:
ocaml/xen-api-client/lwt/dune introduced `ssl`, is it used only by some standalone binary?